mongo/doc.go

```diff
@@ -113,7 +113,8 @@
 //    }
 //    aeo.SetExtraOptions(mongocryptdOpts)
 // To specify a process URI for mongocryptd, the "mongocryptdURI" option can be passed in the ExtraOptions map as well.
-// More information about mongocryptd will soon be available from the official documentation.
+// More information about mongocryptd will soon be available from the official documentation. See the
```

Comment:
I realized our original documentation doesn't mention the build tag so I added it here. Running the examples without specifying the tag results in a panic.